Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
+    {
+        public CollaboratorRequest()
+        {
+            Permission = Permission.Push;
+        }
+
+        public CollaboratorRequest(Permission permissions)
```

Comment:
Since this is a request object, we should put some XmlDoc on this constructor, to give the users intellisense.  Check out `RepositoryPermissionRequest` for an example